config/default/common/config/metadata/layers/goes/GOES-West_ABI_Band2_Red_Visible_1km.md

```diff
@@ -2,6 +2,6 @@ Note: This layer is generally available for the **most recent 90 days**, though
 
 The Red Visible (0.64 um, Band 2) layer from the GOES-West Advanced Baseline Imager (ABI) is used primarily to monitor the evolution of clouds throughout the daylight hours. It is also useful for identifying small-scale features such as river fog/clear air boundaries, or overshooting tops of cumulus clouds. It can also be used to identify daytime snow and ice cover, diagnose low-level cloud-drift winds, assist with detections of volcanic ash and analysis of hurricanes and winter storms.
 
-The Geostationary Operational Environmental Satellites (GOES)-West satellite (currently, GOES-18) is centered on 137.2 degrees W, covering most of the Pacific Ocean, the USA, most of Canada, Central America, the western half of South America, and parts of Australasia. The GOES-West ABI imagery is available on a rolling 90-day basis at 10 minute intervals. The sensor resolution of Band 2 is 0.5 km, the imagery resolution in Worldview/Global Imagery Browse Services (GIBS) is 1 km, the temporal resolution is 10 minutes,  and the latency (time from satellite acqusition to availability in GIBS) is approximtely 40 minutes.
+The Geostationary Operational Environmental Satellites (GOES)-West satellite (currently, GOES-18) is centered on 137.2 degrees W, covering most of the Pacific Ocean, the USA, most of Canada, Central America, the western half of South America, and parts of Australasia. The GOES-West ABI imagery is available on a rolling 90-day basis at 10 minute intervals. The sensor resolution of Band 2 is 0.5 km, the imagery resolution in Worldview/Global Imagery Browse Services (GIBS) is 1 km, the temporal resolution is 10 minutes,  and the latency (time from satellite acquisition to availability in GIBS) is approximtely 40 minutes.
```

Comment:
"approximtely" is misspelled here as well.

Reply:
oh my goodness, I must have been half asleep or something